emission/core/wrapper/compositetrip.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import *
+import emission.core.wrapper.confirmedtrip as ecwc
+import emission.core.wrapper.wrapperbase as ecwb
+import emission.core.wrapper.modeprediction as ecwm
+
+class Compositetrip(ecwc.Confirmedtrip):
+    props = ecwc.Confirmedtrip.props
+    props.update({#      # confirmedplace stuff
+                  "confirmed_place": ecwb.WrapperBase.Access.WORM, # object contains all properties for the destination confirmed_place object
+                  "locations": ecwb.WrapperBase.Access.WORM, # object containing cleaned location entries (max 100)
+                  #      # sections stuff
+                  "cleaned_section": ecwb.WrapperBase.Access.WORM,
+                  "inferred_section": ecwb.WrapperBase.Access.WORM,
+                  "inferred_mode": ecwb.WrapperBase.Access.WORM, # inferred by mode inference algo
+                  "confirmed_mode": ecwb.WrapperBase.Access.WORM, # confirmed by user
+# mode to be used for analysis; confirmed mode if we know factors for it, inferred mode otherwise
```

Comment:
what `factors` do you mean? Also this cannot be the inferred mode because the inferred mode is an array/dict. It can potentially be the primary inferred mode (see https://github.com/e-mission/e-mission-docs/issues/476#issuecomment-738120752)